vkFFT/vkFFT.h

```diff
@@ -281,6 +281,7 @@ typedef struct {
 	hipEvent_t* stream_event;//Filled at app creation
 	uint64_t streamCounter;//Filled at app creation
 	uint64_t streamID;//Filled at app creation
+	int64_t  useStrict32BitAddress; // guarantee 32 bit addresses in bytes instead of number of elements. This results in fewer instructions generated. -1: Disable, 0: Infer based on size, 1: enable. Has no effect with useUint64.
```

Comment:
This will break all other backends as you reference it not only in VKFFT_BBACKEND=2.

Reply:
Yep, seems like I missed a few places, should be fixed now.